### PR TITLE
Pluribus Networks dhcp filter module with unit tests

### PR DESCRIPTION
**Pull request by rajaspachipulusu17**
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
pn_dhcp_filter.py module with unit tests.

##### ADDITIONAL INFORMATION
ansible 2.4.0.0
  config file = None
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]

```

### CODE REVIEW

**Comment by rajaspachipulusu17** [comment]
@Akasurde  can you merge these if you dont have any issues?

Thanks,
Rajas

**Review by pdam-arista — APPROVED** [review]

File: lib/ansible/modules/network/netvisor/pn_dhcp_filter.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  - This module can be used to create, delete and modify a DHCP filter config.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: False
+    type: str
+  state:
+    description:
+      - State the action to perform. Use 'present' to create dhcp-filter and
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_dhcp_filter.py]:
```suggestion
      - State the action to perform. Use C(present) to create dhcp-filter and
```

File: lib/ansible/modules/network/netvisor/pn_dhcp_filter.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        'absent' to delete dhcp-filter 'update' to modify the dhcp-filter.
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_dhcp_filter.py]:
```suggestion
        C(absent) to delete dhcp-filter and C(update) to modify the dhcp-filter.
```

File: lib/ansible/modules/network/netvisor/pn_dhcp_filter.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    required: True
+    choices: ['present', 'absent', 'update']
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_dhcp_filter.py:34]: Type is missing.

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

**Comment by rajaspachipulusu17** [comment]
@Akasurde can you merge this if you dont see any issue?

**Review by Akasurde — APPROVED** [review]

**Pull request merged**